_includes/_guides/_solo_crunching.htm

```diff
@@ -0,0 +1,263 @@
+                  </li>
+                  <li>
+                      Your balance does not affect the amount of the payout,
+                      only how frequent. You need to wait around 16 hours for
+                      received coins to become eligible to earn staking rewards.
+                  </li>
+              </ol>
+
+              <p>
+                  If you don't want to buy any Gridcoin, you can get some through
+                  the faucets or through <a href="#">pool mining</a>.
+                  On most exchanges, you need to buy Bitcoin first in order to
+                  buy Gridcoin.
+              </p>
+          </div><!-- .step-body -->
+      </section><!-- .step -->
+
+      <section class="step">
+          <header>
+              <img src="/assets/img/ic-guide-earn-grc.svg" alt="">
+              <h3>
+                  <span class="echo" aria-hidden="true">07</span>
+                  <strong>Step 7.</strong> Earn Research Rewards
+              </h3>
+          </header>
+          <div class="step-body">
+              <h4 class="h5">Check Your Projects</h4>
+              <ol>
+                  <li>
+                      Make sure any projects you crunch are on the
+                      <a href="/whitelist.htm">whitelist</a>. If a project
+                      isn't approved, you will not be rewarded for it.
+                  </li>
+                  <li>
+                      Check that each project crunched is on team Gridcoin.
+                      You will not earn rewards if they are not on the team.
+                  </li>
+                  <li>
+                      Check that the CPIDs (cross-project identifiers) are the
+                      same across your projects. You can check on each project
+                      or check using
+                      <a href="https://www.boincstats.com/">BOINCstats</a>.
+                      If you found differing CPIDs, use this
+                      <a href="https://steemit.com/gridcoin/@dutch/the-definitive-guide-to-fixing-a-split-cpid-gridcoin-s-and-boinc-s-1-problem">guide</a>.
+                  </li>
+              </ol>
+
+              <h4 class="h5">Configure the Wallet</h4>
+              <ol>
+                  <li>
+                      In the wallet, go to the "overview" page and check the
+                      message in the lower-left. Make sure it doesn't say
+                      "INVESTOR".
+
+                      <ul>
+                          <li>
+                              If it does, ensure that you followed each
+                              of the steps in "Check Your Projects" above.
+                              If you set up BOINC in a non-default path, add
+                              the path to the configuration file (see Step 4).
+                          </li>
+                      </ul>
+                  </li>
+                  <li>Send a beacon.</li>
```

Comment:
This needs a whole step like it had in the more detailed guides. Sending a beacon involves multiple steps

Reply:
This step will require screenshots and a lot of text. This simplified stepper probably not a very good fit for it. It will look very cluttered. I think it deserves a separate page. What if we will have here a link instead to the individual page here instead? 
I can refer to the .us website page temporarily (https://gridcoin.us/guides/earn-grc.htm) and have a ticket to craft the page later as a part of .world website. wdyt?

Reply:
That would work for now